Octokit.Reactive/Clients/IObservableRepositoryContentsClient.cs

```diff
@@ -124,7 +134,19 @@ public interface IObservableRepositoryContentsClient
         /// <returns>
         /// A collection of <see cref="RepositoryContent"/> representing the content at the specified path
         /// </returns>
-        IObservable<RepositoryContent> GetAllContents(string owner, string name, string path, string reference);
+        IObservable<RepositoryContent> GetAllContentsByRef(string owner, string name, string path, string reference);
```

Comment:
I would reorder `path` and `reference`. The reason is the other overload to this method takes in `owner`, `name`, `reference`. If the user is calling that overload, then decides, no, they want to call this one, they only need to tack on the `path` parameter. They don't need to re-order parameters. Based on my experience, this is a common source of API call errors.

Reply:
path parameter is not accepting any special char like '#' and space. I am trying to retrieve files form directory and i am getting not found exception and under same root directory it is getting all file which does not have any special char.
I have two sub directories : 1) C# Code  and 2) TestApp. i am able to access TestApp but not "C# Code" as it is throwing not found exception. suggest me any solution for this as i am not able to rename the directory sue to some reason.